docs/DeveloperGuide.md

```diff
@@ -2,12 +2,134 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
+
+[Original Source](https://github.com/nus-cs2113-AY2425S1/tp)
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Storage
+To load data from previous session:
+Within Wiagi constructor, Storage class is constructed, which will load and initialise incomes, spendings and
+password by de-serialising the text at their distinct file paths. Wiagi will then initialise it incomes and spendings
+based on the member in the Storage class.
+![storageLoad.png](./Diagrams/storageLoad.png)
+
+To save data for current session:
+After the command bye is sent by the user, incomes and spendings will be serialised and overwrite texts in
+their distinct file paths.
+![storageSave.png](./Diagrams/storageSave.png)
+
+### Recurrence Component
+
+#### Motivation behind the component:</br>
++ Allows the user to set specific expenditure and incomes as recurring events to increase efficiency when using the
+  application
++ Users may have differing frequencies for recurring events thus application gives them a few common options
+
+Illustrated below is the class diagram for the Recurrence Component:</br>
+</br>
+<img src="./Diagrams/recurrenceClassDiagram.png" alt="recurrenceClassDiagram" width="800"/>
```

Comment:
![image](https://github.com/user-attachments/assets/678000ef-0d0f-4f32-bfc2-e0373955200c)
parseRecurrence() is a static method as seen later on in the DG, should that method be underlined in the class diagram?